lib/src/_shared/icons/ArrowDropDownIcon.tsx

```diff
@@ -1,4 +1,4 @@
 import React from 'react';
-import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';
+import createSvgIcon from '@material-ui/core/utils/createSvgIcon';
```

Comment:
```suggestion
import { createSvgIcon } from '@material-ui/core/utils';
```

I wish it was solved in the different pull request!